common/src/commonMain/kotlin/com/studo/campusqr/common/payloads/AccessManagementPayloads.kt

```diff
@@ -0,0 +1,56 @@
+package com.studo.campusqr.common.payloads
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+class AccessManagementData(
+  val accessManagement: List<ClientAccessManagement>,
+  val clientLocation: ClientLocation?,
+) : ClientPayload
+
+@Serializable
+class AccessManagementExportData(
+  val permits: List<Permit>,
+  val clientLocation: ClientLocation?,
+) : ClientPayload {
+  @Serializable
+  class Permit(
+    val dateRange: ClientDateRange,
+    val email: String,
+  )
+}
+
+@Serializable
+class ClientAccessManagement(
+  val id: String,
+  val locationName: String,
+  val locationId: String,
+  val allowedEmails: List<String>,
+  val dateRanges: List<ClientDateRange>,
+  val note: String,
+  val reason: String,
+) : ClientPayload
+
+@Serializable
+class ClientDateRange(
```

Comment:
these two have the Client- prefix while the others don't. Should we unify this or is there a reason for it?

Reply:
Server uses Backend- prefix for the corresponding classes